docs/specifications/tests/Address-TP/address01.md

```diff
@@ -92,7 +43,7 @@ forbidden blocks, the test case succeeds.
 
 ### Special procedural requirements
 
-None.
+Should automatically fetches the registry mentioned above when need for testing 
 
```

Comment:
The language needs some correction and clarification. Suggestion: "The registries listed in bullet 3 must be fetched prior to testing."